libs/npm/core/package.json

```diff
@@ -41,7 +41,7 @@
     "react-table": "^7.8.0",
     "react-text-mask": "^5.5.0",
     "react-toastify": "^9.1.1",
-    "react-tooltip": "^4.5.1",
+    "react-tooltip": "5.9.0",
```

Comment:
This needs to be a `^5.9.0` as it's a package and not the project, otherwise it causes other kinds of issues.